spec/integration/users_index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Users#index', type: :system do
+  before(:all) do
+    Comment.delete_all
+    Post.delete_all
+    User.delete_all
+    @tom = User.create(name: 'Tom', photo: 'https://placehold.co/200x133', bio: 'Teacher from Mexico.',
+                       posts_counter: 0)
+    Post.create(author: @tom, title: 'Hello', text: 'This is my first post', comments_counter: 0, likes_counter: 0)
+    @lilly = User.create(name: 'Lilly', photo: 'https://placehold.co/200x133', bio: 'Teacher from Poland.',
+                         posts_counter: 0)
+    @users = User.all
+  end
+
+  it 'I can see the username of all other users.' do
+    visit '/users'
+    expect(page).to have_content('Tom')
+    expect(page).to have_content('Lilly')
+  end
```

Comment:
- [ ] This test and the other, as shown in the attached test result are failing, I will encourage to check it again
![image](https://github.com/Bifabig/blog-app-rails/assets/86200356/b5bf9274-d2e0-4edc-8e1d-f99fb8c729d9)